bin/porta

```diff
@@ -260,7 +278,7 @@ module Porta
     def initialize
       super
 
-      return if ARGV[1] && !ARGV[1].start_with?('-')
+      return if ARGV[0] && !ARGV[0].start_with?('-')
```

Comment:
Why this? `TestOptionParser` inherits from `RailsEnvOptionParser` and that one didn't change.

Reply:
It is related to this change:
https://github.com/3scale-labs/porta-dev-tools/pull/19/files#diff-422e0549dec116dd1a61191fce586358e3ec8f56ebbe702d29b738ffdbe22a5cR1062

When using `shift` the original first argument is gone from the list at this point.